docs/se/guides/overview.adoc

```diff
@@ -67,6 +67,41 @@ Learn how to use Helidon SE built-in and application metrics.
 --
 Learn how to trace a Helidon SE application.
 --
+
+[CARD]
+.OIDC Guide
+[link=security-oidc.adoc]
+--
+Learn how to set up OIDC Helidon SE application.
```

Comment:
OIDC -> an OIDC
Drop "." at end for consistency?

Reply:
Done!